test-interface/src/main/scala/scala/scalanative/testinterface/TestMain.scala

```diff
@@ -31,53 +31,27 @@ object TestMain {
   final val iPv4Loopback = "127.0.0.1"
   final val iPv6Loopback = "::1"
```

Comment:
This val seems to be unused, probably for a long time.

Although it goes against my usual "don't change unless the change is __needed_ for __this__ PR" approach,
I think deleting the line (if I am correct about it being unused), helps a maintainer trace if and how IPv6 
might be being used.  

On known IPv6 systems, using the IPv6Loopback is likely to be slightly more efficient. I had thought that
once upon a time, this file did so.

Reply:
See comment above, I'll remove both variables and restore the direct use of `127.0.0.1`